randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -692,7 +702,12 @@ Extra - asset_id: 92
   * Layers: default
   * Normal Door to Noob Bridge/Door to Green Hill Zone
   > Door to Big Pink
-      Wave Beam
+      Any of the following:
+          Wave Beam
+          All of the following:
+              # Gate Glitch
+              Gate Glitch (Beginner)
+              Missile ≥ 5 or Super Missile ≥ 5
```

Comment:
can you not do this with only one (super) missile?

Reply:
You could, but one would like a little breathing room for the trick. Its not trivial and usually takes a handful of shots.

Reply:
hmm, i'm wondering mostly for runs that reduce the amount of ammo. maybe have beginner with 5 and a harder difficulty with only 1 ammo?

Reply:
Done.